src/transformers/pipelines/image_segmentation.py

```diff
@@ -105,6 +110,9 @@ def __call__(self, images, **kwargs) -> Union[Predictions, List[Prediction]]:
 
                 The pipeline accepts either a single image or a batch of images. Images in a batch must all be in the
                 same format: all as HTTP(S) links, all as local paths, or all as PIL images.
+            task_inputs (`List[str]`):
+                The value of task token inputs for task-dynamic inference, choose [`semantic`, `instance` or
+                `panoptic`]. task_inputs is only needed for [`~OneFormerForUniversalSegmentation`]
```

Comment:
Pinging @Narsil here to add support for OneFormer for the image segmentation pipeline.

For context, OneFormer is the only model that requires an additional `task_inputs` to perform image segmentation